spec/tapioca/compilers/dsl/url_helpers_spec.rb

```diff
@@ -158,9 +158,10 @@ class Application < Rails::Application
 
       expected = <<~RBI
         # typed: strong
+
         module GeneratedUrlHelpersModule
-          include ::ActionDispatch::Routing::PolymorphicRoutes
           include ::ActionDispatch::Routing::UrlFor
+          include ::ActionDispatch::Routing::PolymorphicRoutes
```

Comment:
This is now in the same order we generate them:
![image](https://user-images.githubusercontent.com/583144/125708532-9ccf9a68-9d5f-40c4-b383-a4dd4a864e12.png)

Not sure why Parlour was changing the order?

Reply:
Nice! Thankfully, the order didn't matter here.